### PR TITLE
Enable Australian CMP

### PR DESCRIPTION
**Pull request by GHaberis**
## What are you doing in this PR?

This PR upgrades the `@guardian/consent-management-platform` dependency `^6.6.1` and updates the way we call `cmp.init()`.

[**Trello Card**](https://trello.com/c/KtOBEhBR/3442-upgrade-cmp-library-to-v661-on-support-frontend)

## Why are you doing this?

There’s a new opt-out consent framework for Australia. It’s likely to be launched Thursday 12/11/2020. 

The new initialisation requires passing in the ISO country code as a string instead of the `isInUsa` boolean, we should therefore update to the latest version of the `@guardian/consent-management-platform` (`v6.6.1`).

## Screenshots

Screenshot 2020-11-16 at 10 28 12

### CODE REVIEW

**Comment by GHaberis** [comment]
@guardian/commercial-dev preparation PR for the Australian CMP.

**Review by mxdvl — APPROVED** [review]
*STILL AWAITING CONFIRMATION TO MERGE*

Great!

**Comment by mxdvl** [comment]
We've released a fix in 6.7.3 which should fix the CORS issues.

File: support-frontend/yarn.lock [hunk 1]
```diff
@@ -2862,10 +2862,10 @@
     camelcase "^5.0.0"
     prettier "^1.13.7"
 
-"@guardian/consent-management-platform@^6.1.0":
-  version "6.1.1"
-  resolved "https://registry.yarnpkg.com/@guardian/consent-management-platform/-/consent-management-platform-6.1.1.tgz#1d3170d569f748d506a134e7261f1d795aceda6d"
-  integrity sha512-uha2dvmlSHj2jhPxdzxBSliaXDT0vFQ2X6Jk2GbHWWyD6rT9PWDhthOtuLtmrkKlSpDU9wG8LuKD3vN/EClXDw==
+"@guardian/consent-management-platform@^6.7.3":
```
Comment (mxdvl) [inline support-frontend/yarn.lock:2865]: This seems slightly out of date?

**Pull request merged**